namespaces/cloud-platform-test-1.k8s.integration.dsd.io/traefik/README.md

```diff
@@ -0,0 +1,80 @@
+# Traefik ingress controller
+
+## Introduction
+Traefik is a very powerful ingress controller that is easy to implement and manage.
+One of it's highlight features is that it offers a dashboard that visualises your hosts and offers health checks.
+
+This guide will serve the purpose of installing Traefik on a cluster and also how to provision ingress objects that run through Treafik.
+
+## Installation
+This guide will use a collection of manifest files to install Traefik.
+
+In this directory, you will see three manifest files, `traefik-deployment.yaml`, `traefik-rbac.yaml`, and `traefik-ui.yaml`.
+
+
+### Route 53 Config
```

Comment:
This is not required as the setup is much easier achieved in the same way nginx-ingress works. See comments below.